src/server/src/main/docker/Dockerfile

```diff
@@ -71,45 +71,51 @@ ENV REAPER_SEGMENT_COUNT=200 \
     REAPER_SHIRO_INI="" \
     JAVA_OPTS=""
 
+# used to run spreaper cli
+RUN apk add --update \
+    python \
+    py-pip \
+  && pip install requests \
+  && rm -rf /var/cache/apk/*
+
+
 ADD cassandra-reaper.yml /etc/cassandra-reaper.yml
 ADD shiro.ini /etc/shiro.ini
 ADD entrypoint.sh /usr/local/bin/entrypoint.sh
 ADD configure-persistence.sh /usr/local/bin/configure-persistence.sh
 ADD configure-metrics.sh /usr/local/bin/configure-metrics.sh
 ADD configure-webui-authentication.sh /usr/local/bin/configure-webui-authentication.sh
 ADD ${SHADED_JAR} /usr/local/lib/cassandra-reaper.jar
+ADD spreaper /usr/local/bin/spreaper
 
-# get around `/usr/local/bin/configure-persistence.sh: line 65: can't create /etc/cassandra-reaper.yml: Interrupted system call` unknown error
-RUN touch /etc/cassandra-reaper.yml
-
-RUN mkdir -p /var/lib/cassandra-reaper && \
+RUN addgroup -g 1001 -S reaper && adduser -G reaper -S -u 1001 reaper && \
+    mkdir -p /var/lib/cassandra-reaper && \
```

Comment:
@velo, could we keep the `touch` on the yaml file?
We've seen that this could prevent the docker image from working properly under some circumstances, especially when you set it up with a storage backend or enable shiro authentication.
Any specific reason why you removed it?